schema.ts

```diff
@@ -756,18 +757,22 @@ export interface ObjectSchemaDefinition<K extends string, L extends string> exte
   type: ValueType.Object;
   /** Definintion of the key-value pairs in this object. */
   properties: ObjectSchemaProperties<K | L>;
+  /** @deprecated Use {@link idProperty} */
```

Comment:
this `link` didn't work for me. I never figured out how to link a property within the same class.

Reply:
Will look at this.

Reply:
Hmm, this does seem to work for me as-is in the staged documentation? I think I did read that the link resolution strategy does try to resolve to properties on the same object first.

Reply:
right it seems to work here. i am a bit confused. I'll see if I hit the issue next time. sorry for the false alarm.